kuzzleio/sdk-javascript
# Choose GET route by default in case of double route

## What does this PR do?

This PR changes the way we get the correct route if there is several of them.

Before this PR:
The SDK checked the size of the url, and chose the shortest.
If they had the same size, it chose the POST route.
And you could not force the verb in your request.

Now:
Same thing, except it chooses the GET route.
Plus, you can force the verb with the option `{verb: 'POST'}`




### How should this be manually tested?

Install this version of the sdk `npm install git://github.com/kuzzleio/sdk-javascript#handle-double-routes`
Run Kuzzle in debug mod.
Run this script
```
const { Kuzzle, Http } = require('kuzzle-sdk');

const kuzzle = new Kuzzle(
  new Http(`localhost`, { port: 7512})
);

async function main () {
const doc1 = { capacity: 4 };
const doc2 = { capacity: 7 };

    try {
	await kuzzle.connect();
	await kuzzle.document.createOrReplace('nyc-open-data', 'yellow-taxi', 'some-id', doc1);
	await kuzzle.document.createOrReplace('nyc-open-data', 'yellow-taxi', 'some-other-id', doc2);

  const response = await kuzzle.document.mGet(
    'nyc-open-data',
    'yellow-taxi',
      ['some-id', 'some-other-id']
//   ,{verb: 'POST'}
  );

  console.log(response);

} catch (error) {
  console.error(error.message);
}
}

main()

```

You will see in the logs that the route verb was GET

Now uncomment the line `//   ,{verb: 'POST'}`
The route will be in POST.

You can do the same with `mGetUsers`

### Other changes

Adapt doc snippet test for `getAllStats` and `getLastStats`, because of the new properties order of the response object [recently defined](https://github.com/kuzzleio/kuzzle/pull/1561/files#diff-ed4e16c2c6915728845ce78457bf7c9dL44)



### Boyscout
